src/bosh_azure_cpi/lib/cloud/azure/storage_account_manager.rb

```diff
@@ -222,7 +225,7 @@ def default_storage_account
       @logger.debug("Can't find a storage account with the tags `#{STEMCELL_STORAGE_ACCOUNT_TAGS}'")
       @logger.debug("Will look for the old storage account (with the table `#{STEMCELL_TABLE}') which stores all uploaded stemcells")
       storage_account = storage_accounts.find do |s|
-        s[:account_type].downcase.start_with?('standard') && has_stemcell_table?(s[:name])
+        s[:sku_tier] == 'Standard' && has_stemcell_table?(s[:name])
```

Comment:
better to do the cowncase compare.

Reply:
Sure. Will change back to the downcase compare.